WTWSMS/decisions/LI_target_decisions.txt

```diff
@@ -1,4 +1,4 @@
-targetted_decisions = {	
+decisions = {	
```

Comment:
Dude, I won;t be able to go to the Forums for another six months, and after that, I will still be punished for another two years on "Probation", in which I can view (but not post) in the Mods and AAR section.

I have tried reporting the glitch where I cannot see the Forum, check my private messages, or even log out before, but I never get a reply. Banning means I cannot post on the Forum. It does not mean I cannot view my private messages and log out of the Forum.. Something has been wrong for a long time on my end.